docs/whats_new.rst

```diff
@@ -18,6 +18,15 @@ Breaking Changes
 
 Enhancements
 ~~~~~~~~~~~~
+- Works with unstructured 1D grids such as:
+
+  - `ICON <https://code.mpimet.mpg.de/projects/iconpublic>`_
+  - `FESOM <https://fesom.de/>`_
+  - `MPAS <https://mpas-dev.github.io/>`_
+
+  with variable ``var(cell)`` and coordinates ``lon(cell)`` and ``lat(cell)``
```

Comment:
```suggestion
  with coordinates of the form ``lon(cell)`` and ``lat(cell)``
```

Reply:
I wanted to explicitly show that the data vars only have this one spatial dimension